### DISCUSSION TITLE
Is data edit (cell) a feature you would add?

### DISCUSSION THREAD

**Discussion by wuarmin** [table-library/react-table-library#22] [Q&A]
What do you think about an editing feature/cookbook? What do you have to consider? What are the challenges? Or should it be straightforward?

Thanks and best regards

**Comment by rwieruch** [comment]
Just that I understand you here: You mean a feature where you are able to toggle a cell and edit its content?

**Reply by wuarmin** [reply]
Yes exactly that.
